clause/limit_test.go

```diff
@@ -8,18 +8,26 @@ import (
 )
 
 func TestLimit(t *testing.T) {
+	limit0 := 0
```

Comment:
test cases should not be changed unless for a reason that's good enough, this is not good enough.

Reply:
+1